db/migrate/20200918101304_data_rb_users_techsource_account.rb

```diff
@@ -0,0 +1,9 @@
+class DataRbUsersTechsourceAccount < ActiveRecord::Migration[6.0]
+  def up
+    User.responsible_body_users.update(orders_devices: true)
```

Comment:
this will trigger the code that generates `UserChange` entries, right?

Reply:
it will. is that what we want?

Reply:
yes, definitely.